end_to_end/tpu/test_sft_trainer.sh

```diff
@@ -49,15 +49,15 @@ FINE_TUNED_MODEL_CKPT_PATH=${CHECKPOINTS_PATH}/${largest_dir}/items
 PROMPT="<user>${PROMPT}</user> <assistant>"
 
 # Decode
-python MaxText/decode.py MaxText/configs/sft.yml \
+python3 -m MaxText.decode MaxText/configs/sft.yml \
     run_name=${RUN_NAME}-hf-decode \
     model_name=${PRE_TRAINED_MODEL} tokenizer_path=${PRE_TRAINED_MODEL_TOKENIZER} \
     load_parameters_path=${FINE_TUNED_MODEL_CKPT_PATH} \
     per_device_batch_size=${PER_DEVICE_BATCH_SIZE} max_target_length=128 max_prefill_predict_length=64 \
     attention=dot_product decode_sampling_strategy=greedy prompt="${PROMPT}"
 
 # Correctness test
-python MaxText/tests/sft_trainer_correctness.py \
+python3 MaxText/tests/sft_trainer_correctness.py \
```

Comment:
This doesn't need a similar change ?

Reply:
All throughout the codebase I try and have consistent usage of: `python3` > `python`; `python3 -m pip` > `pip` | `pip3`; `python3 -m <module>` > `python3 <script_file>.py`